scripts/tasks/codepen-examples-transform.js

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
As we mentioned in standup, it would be really good to have a test for this transform which could serve as a documentation on what a "good" example would be like.